lib/cloudwatch-integration.js

```diff
@@ -128,6 +144,11 @@ lib.getToken = function(aws, groupName, streamName, retentionInDays, cb) {
     }
   });
 };
+lib.getToken.lastNextToken = {};
```

Comment:
I would suggest to have this as a variable in this module, rather than attached to the function. Do you have any particular reason why you did it this way?